README.md

```diff
@@ -1,70 +1,166 @@
+
+- GitHub: [@ahzamir](https://github.com/ahzamir).
+- Twitter: [@YousufiZamir](https://twitter.com/YousufiZamir)
+- LinkedIn: [ahzamir](https://www.linkedin.com/in/ahzamir/)
+
+👤 **Huseyin Biyik** <img src="https://emojis.slackmojis.com/emojis/images/1531849430/4246/blob-sunglasses.gif?1531849430" width="20"/>
+
+- GitHub: [@huseyinbiyik](https://github.com/huseyinbiyik).
+- Twitter: [@huseyin__biyik](https://twitter.com/huseyin__biyik)
+- LinkedIn: [Huseyin_biyik](http://www.linkedin.com/in/tahahuseyinbiyik/)
+
+👤 **Precious Akams** <img src="https://emojis.slackmojis.com/emojis/images/1531849430/4246/blob-sunglasses.gif?1531849430" width="20"/>
+
+- GitHub: [@kamzzy](https://github.com/kamzzy).
+- Twitter: [@precious_akams](https://twitter.com/precious_akams)
+- LinkedIn: [Onyeagoziri_Akams](https://www.linkedin.com/in/Onyeagoziri_Akams/)
+
+👤 **Felix Odete** <img src="https://emojis.slackmojis.com/emojis/images/1531849430/4246/blob-sunglasses.gif?1531849430" width="20"/>
+
+- GitHub: [@felixodette](https://github.com/felixodette).
+- Twitter: [@Odette3000](https://twitter.com/Odette3000)
+- LinkedIn: [Felix_Odete](https://www.linkedin.com/in/Felix_Odete/)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+Feel free to check the [issues page](https://github.com/felixodette/hotel-room-frontend/issues).
 
-### `npm run build`
+### Attributions and Credit
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+Orignal design idea by [Murat Korkmaz on Behance](https://www.behance.net/muratk).
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+## Show your support
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+Give a ⭐️ if you like this project!
 
-### `npm run eject`
+## Acknowledgments
 
-**Note: this is a one-way operation. Once you `eject`, you can't go back!**
+- [Microverse](https://www.microverse.org/)
+- Code Reviewers
+- Coding Partners
 
-If you aren't satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+## 📝 License
 
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you're on your own.
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] This license link is broken, because it points to a file that does not exist.  Kindly fix it so the license doc appears when the link is clicked.

Reply:
Thank you so much for the feedback, Miguel. I will handle this issue.